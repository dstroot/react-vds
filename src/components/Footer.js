import React from 'react';
import { NavLink } from 'react-router-dom';

const Footer = () => {
  const year = new Date(); // evergreen copyright year
  return (
    <footer className="bg-gray-lighter text-secondary mt-4">
      <hr />
      <div className="container font-weight-light">
        <div className="row py-3">
          <div className="col-sm-8">
            <h4>About Us</h4>
            <p className="my-2">
              We support the digital journey of employees, data, software, and
              infrastructure to cloud-native technologies.
            </p>
          </div>
          <div className="col-sm-4">
            <h4>Links</h4>
            <ul className="list-unstyled">
              <li>
                <NavLink to="/contact">Contact Us</NavLink>
              </li>
              <li>
                <NavLink to="/privacy">Privacy</NavLink>
              </li>
              <li>
                <NavLink to="/terms">Terms of Use</NavLink>
              </li>
            </ul>
          </div>
        </div>
        <div className="row">
          <div className="col-sm-8">
            <p className="small">
              Copyright Â© {year.getFullYear()} Verite Data Science, LLC, all
              rights reserved.
            </p>
          </div>
        </div>
      </div>
    </footer>
  );
};

export default Footer;
