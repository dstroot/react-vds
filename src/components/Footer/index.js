import React from 'react';
import { Link } from '@reach/router';
import * as routes from '../../constants/routes';

const Footer = () => {
  const year = new Date(); // evergreen copyright year

  return (
    <footer className="bg-gray-lighter text-secondary mt-auto py-3">
      <div className="container font-weight-light">
        <div className="row py-3">
          <div className="col-sm-8">
            <h4>About Us</h4>
            <p className="my-2">
              We support the digital journey of employees, data, software, and
              infrastructure to cloud-native technologies.
            </p>
          </div>
          <div className="col-sm-4">
            <h4>Links</h4>
            <ul className="list-unstyled">
              <li>
                <Link to={routes.CONTACT}>Contact Us</Link>
              </li>
              <li>
                <Link to={routes.PRIVACY}>Privacy</Link>
              </li>
              <li>
                <Link to={routes.TERMS}>Terms of Use</Link>
              </li>
            </ul>
          </div>
        </div>
        <div className="row">
          <div className="col-sm-8">
            <p className="small">
              Copyright Â© {year.getFullYear()} Verite Data Science, LLC, all
              rights reserved.
            </p>
          </div>
        </div>
      </div>
    </footer>
  );
};

export default Footer;
