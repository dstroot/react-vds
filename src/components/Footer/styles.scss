//
// Footer
//

footer {
  // background-color: hsla(0, 0, 0, 0.045);
  a {
    color: theme-color('primary');
    &:hover,
    &:focus {
      color: theme-color('special');
      text-decoration: none;
    }
  }

  // space links to pass lighthouse audit
  li {
    padding-bottom: 5px;
  }
}
