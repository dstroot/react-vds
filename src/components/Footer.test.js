import React from 'react';
import Footer from './Footer';
import { BrowserRouter as Router } from 'react-router-dom';
import renderer from 'react-test-renderer';

test('Footer renders a footer', () => {
  const component = renderer.create(
    <Router>
      <Footer />
    </Router>
  );

  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
