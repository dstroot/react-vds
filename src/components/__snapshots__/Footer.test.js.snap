// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer renders a footer 1`] = `
<footer
  className="bg-gray-lighter text-secondary mt-auto py-3"
>
  <hr />
  <div
    className="container font-weight-light"
  >
    <div
      className="row py-3"
    >
      <div
        className="col-sm-8"
      >
        <h4>
          About Us
        </h4>
        <p
          className="my-2"
        >
          We support the digital journey of employees, data, software, and infrastructure to cloud-native technologies.
        </p>
      </div>
      <div
        className="col-sm-4"
      >
        <h4>
          Links
        </h4>
        <ul
          className="list-unstyled"
        >
          <li>
            <a
              aria-current={null}
              className={undefined}
              href="/contact"
              onClick={[Function]}
              style={undefined}
            >
              Contact Us
            </a>
          </li>
          <li>
            <a
              aria-current={null}
              className={undefined}
              href="/privacy"
              onClick={[Function]}
              style={undefined}
            >
              Privacy
            </a>
          </li>
          <li>
            <a
              aria-current={null}
              className={undefined}
              href="/terms"
              onClick={[Function]}
              style={undefined}
            >
              Terms of Use
            </a>
          </li>
        </ul>
      </div>
    </div>
    <div
      className="row"
    >
      <div
        className="col-sm-8"
      >
        <p
          className="small"
        >
          Copyright Â© 
          2018
           Verite Data Science, LLC, all rights reserved.
        </p>
      </div>
    </div>
  </div>
</footer>
`;
